replacer/templates-generator.mjs

```
@@ -102,17 +106,26 @@ function appendToConfig(configFilePath, replaces, appendLineFunc) {
   const startIndex = replace.replace.indexOf('{{');
   const finalIndex = replace.replace.indexOf('}}');
   if(startIndex >= 0 && finalIndex >= 0){
-    replace.replace = replace.replace.substring(startIndex, finalIndex + 2);
+    replace.replace = replace.replace.substring(startIndex +2 , finalIndex);
     return true
   }
   return false
 }
+//config is the last word splited by spaces
+function leaveOnlyConfig(replace){
+  if(replace && replace.replace)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)